src/main/java/com/hillel/items_exchange/controller/AdvertisementController.java

```diff
@@ -114,6 +114,22 @@
         return ResponseEntity.unprocessableEntity().body(HttpStatus.CONFLICT);
     }
 
+    @PostMapping("/default-image/{advertisementId}/{imageId}")
+    public ResponseEntity<HttpStatus> setDefaultImage(
+            @PathVariable @PositiveOrZero(message = "{invalid.id}") Long advertisementId,
+            @PathVariable @PositiveOrZero(message = "{invalid.id}") Long imageId,
+            Principal principal) {
+        User owner = getUser(principal.getName());
+
+        try {
+            advertisementService.setDefaultImage(advertisementId, imageId, owner);
+        } catch (ClassNotFoundException e) {
+            log.warn(e.getMessage());
```

Comment:
Add the exception to log for the stacktrace getting: `log.warn(e.getMessage(), e);`